### ISSUE TITLE
Refactor: url_list JavaScript file

### ISSUE THREAD

**Issue by dhanur-sharma** [NASA-IMPACT/sde-indexing-helper#1110]
## Description
The delta_url_list.js is very big and needs to be modularized for ease of development updates.

For starters, just add the improved column implementation from the collection list page. No need to do a full refactor if you can easily do the improved column editing. However, make another card for the full refactor.

## Dependencies
depends on #

``` [tasklist]
### Acceptance Criteria
- [ ] Criteria 1
```

**Comment by CarsonDavis** [comment]
This is not a priority for 25.1. We can move it to 25.2.

**Comment by bishwaspraveen** [comment]
```
columnDefs: [
    // hide the data columns
    {
      targets: [COLUMNS.WORKFLOW_STATUS_RAW, COLUMNS.CURATOR_ID, COLUMNS.REINDEXING_STATUS_RAW, COLUMNS.REINDEXING_CURATOR_ID],
      visible: false, width: "0px", responsivePriority: -1
    },
``` 
Take a look at this piece of code and see if this can be used efficiently used in other places, specifically responsivePriority.

**Comment by shravan-lrm** [comment]
@dhanur-sharma @advaityogaonkar-lrm is this still relevant , if not can we close this?
